blockbuster/lanes/auction/lane.go

```diff
@@ -0,0 +1,58 @@
+package auction
+
+import (
+	"github.com/cometbft/cometbft/libs/log"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/skip-mev/pob/blockbuster"
+)
+
+const (
+	// LaneNameTOB defines the name of the top-of-block auction lane.
+	LaneName = "tob"
+)
+
+var _ blockbuster.Lane = (*TOBLane)(nil)
+
+// TOBLane defines a top-of-block auction lane. The top of block auction lane
+// hosts transactions that want to bid for inclusion at the top of the next block.
+// The top of block auction lane stores bid transactions that are sorted by
+// their bid price. The highest valid bid transaction is selected for inclusion in the
+// next block. The bundled transactions of the selected bid transaction are also
+// included in the next block.
+type TOBLane struct {
+	// Mempool defines the mempool for the lane.
+	Mempool
+
+	// LaneConfig defines the base lane configuration.
+	*blockbuster.LaneConfig
```

Comment:
1. Why do we embed?
2. Why do we expect a pointer?

```suggestion
	// LaneConfig defines the base lane configuration.
	cfg blockbuster.LaneConfig
```

Reply:
I think all of these are going to be internal methods anyways so i'll update it so that we do not embed. 

re pointer: no reason ill change it to be a standard type.